meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-atf/imx-atf_%.bbappend

```diff
@@ -1,7 +1,8 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
 
-# lf-5.10.52-2.1.0
-SRCREV = "bb4957067d4b96a6ee197a333425948e409e990d"
+SRCBRANCH = "lf_v2.4"
```

Comment:
I don't understand why this recipe is the only one that the branch is an issue. Would you, please, elaborate a bit? (related with the commit log, but I don't know how to comment near the commit log)

Reply:
Well, actually with meta-freescale buming it doesn't need, but generally the problem was that nxp rewrote its recipe and changed the default branch so that the commit we'd used before in this recipe (bb495706) disappeared.
So that sticking branch alongside with a commit is more reliable solution than just a commit hash.
It's all ASAIRC, I can recheck the reason tomorrow.